cfgov/v1/forms.py

```diff
@@ -277,6 +277,28 @@ class EnforcementActionsFilterForm(FilterableListForm):
         widget=widgets.CheckboxSelectMultiple()
     )
 
+    products = forms.MultipleChoiceField(
+        required=False,
+        choices=enforcement_action_page.enforcement_products,
+        widget=widgets.SelectMultiple(attrs={
+            'id': 'o-filterable-list-controls_products',
+            'class': 'o-multiselect',
+            'data-placeholder': 'Search for products',
+            'multiple': 'multiple',
+        })
+    )
+
+    at_risk_groups = forms.MultipleChoiceField(
+        required=False,
+        choices=enforcement_action_page.enforcement_at_risk_groups,
```

Comment:
Should the choices in the dropdown be in a certain order? They appear random currently.

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/6241